tests/test_utils.py

```diff
@@ -63,11 +73,14 @@ def setUp(self):
             self.exposure.getBBox(),
         )
         self.reference_pipeline = make_test_reference_pipeline()
+        self.injected_catalog = self.injection_catalog.copy()
+        self.injected_catalog.add_columns(cols=[0, 0], names=["injection_draw_size", "injection_flag"])
```

Comment:
I guess these should always be in the `injection_catalog`, right? In which case, you can move this logic into `utils/test_utils.py` in the `make_test_injection_catalog` function. Is that correct?

Reply:
PS - apologies for not catching this yesterday. I only saw the key-word `injected_` yesterday, so assumed it was a new column. But, this is actually an `injection_` column, so should probably live in the main `make_test_injection_catalog` constructor.

Reply:
`make_test_injection_catalog` makes an `injection_catalog` which (for a catalog of all point sources) should only have the columns`['injection_id','ra','dec','mag','source_type']` right? I am waiting to add `injection_draw_size` and `injection_flag` until I make the simulated `injected_deepCoadd_catalog`. I think I am misunderstanding something.

Reply:
Ah, no, quite right! I was again mixing up my injected with my injection. Ok, in which case, I think what you have here is just fine!